parm/use_cases/met_tool_wrapper/TCGen/TCGen.conf

```diff
@@ -104,13 +104,19 @@ TC_GEN_DLAND_THRESH = NA
 
 TC_GEN_GENESIS_MATCH_RADIUS = 500
 
+#TC_GEN_GENESIS_MATCH_POINT_TO_TRACK = True
+
+#TC_GEN_GENESIS_MATCH_WINDOW_BEG = 0
+#TC_GEN_GENESIS_MATCH_WINDOW_END = 0
+
+#TC_GEN_OPS_HIT_WINDOW_BEG = 0
+#TC_GEN_OPS_HIT_WINDOW_END = 48
+
```

Comment:
I think these should be uncommented

Reply:
These were left commented intentionally. The idea is that the basic met_tool_wrapper use case lists all of the possible config variables that can be used by the wrapper, but do not actually set the optional values. These are here for reference so users can easy see what they can change. If these were uncommented and set to the default value that corresponds to the default values in TCGenConfig_default, then we would have to update this use case conf every time a default value changes on the MET side. I use these variables to test that the values read into the MET config are read properly and don't cause the tool to crash, then comment them out for the final version.

Reply:
Thanks!